systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,357 @@
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername, 240000))
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.defaultAdmin().build());
+        String createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(
+                createPodName,
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                GLOBAL_TIMEOUT
+        );
+        JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+
+        // Teardown delete created topics
+        KafkaTopicUtils.deleteAllKafkaTopicsWithPrefix(NAMESPACE, topicNamePrefix);
+    }
+
+    @ParallelTest
+    void testThrottlingQuotasDeleteTopic(ExtensionContext extensionContext) {
+        final String kafkaUsername = mapWithTestUsers.get(extensionContext.getDisplayName());
+        final String createAdminName = "create-admin-" + mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        final String deleteAdminName = "delete-admin-" + mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        final String topicNamePrefix = classTopicPrefix + "-delete";
+        int topicsCountOverQuota = 500;
+        setupKafkaUserInNamespace(extensionContext, kafkaUsername);
+
+        // Create many topics in multiple rounds using starting offset
+        KafkaAdminClient createAdminClientJob;
+        String createPodName;
+        int offset = 100;
+        int iterations = topicsCountOverQuota / 100;
+        for (int i = 0; i < iterations; i++) {
+            LOGGER.info("Executing {}/{} iteration.", i + 1, iterations);
+            createAdminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(100)
+                .withTopicOffset(i * offset)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername, 240000))
+                .build();
+            resourceManager.createResource(extensionContext, true, createAdminClientJob.defaultAdmin().build());
+            ClientUtils.waitForClientSuccess(createAdminName, NAMESPACE, 100);
+            createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+            PodUtils.waitUntilMessageIsInPodLogs(createPodName, "All topics created", Constants.GLOBAL_TIMEOUT);
+            JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        }
```

Comment:
How I mentioned in previous few comments, you can create some default job and then just use `toBuilder()` and change everything needed (which in this case is one field)